go/config/defaults/0-base-config.yaml

```diff
@@ -0,0 +1,91 @@
+# Base config for all TEN services
+#  - this contains sensible defaults where possible
+#  - serves as template/documentation for config structure
+
+network:
+  chainId: 443
+  genesis: "{}" # json string of genesis block, used for configuring prefunded accounts etc.
```

Comment:
Do we still rely on genesis override in sim tests? Perhaps a good opportunity to have test/1-sim.yaml that showcases overrides